toolkit/tools/pkg/imagecustomizerlib/customizebootloader.go

```diff
@@ -14,12 +14,17 @@ import (
 )
 
 func handleBootLoader(baseConfigPath string, config *imagecustomizerapi.Config, imageConnection *ImageConnection,
-	partUuidToFstabEntry map[string]diskutils.FstabEntry,
+	partUuidToFstabEntry map[string]diskutils.FstabEntry, newImage bool,
 ) error {
+	if newImage && config.OS.BootLoader.ResetType != imagecustomizerapi.ResetBootLoaderTypeHard {
+		logger.Log.Warnf(
```

Comment:
We don't use warnings unless it is something the user must fix. Messages that may or may not be a problem are put at INFO or lower.